api/codelists.go

```diff
@@ -0,0 +1,89 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/ONSdigital/dp-code-list-api/models"
+	"github.com/ONSdigital/log.go/log"
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+)
+
+func (c *CodeListAPI) getCodeLists(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	filterBy := r.URL.Query().Get("type")
+
+	log.Event(ctx, "attempting to get code lists", log.INFO, log.Data{"type": filterBy})
+
+	dbCodeLists, err := c.store.GetCodeLists(r.Context(), filterBy)
+	if err != nil {
+		log.Event(ctx, "failed to get code lists from graph", log.INFO, log.Data{"err": err, "type": filterBy})
```

Comment:
This should be `log.ERROR`, with `log.Error(err)`, instead of it being in the log.Data?

Reply:
in fact, handleError also logs the error, with ERROR level. I will remove this log, and accept a message in handleError